### DISCUSSION TITLE
Add default "Save and stay" option to all "settings" pages

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#14168] [Ideas] [answered]
Currently section edit page has such an option but for example site edit page does not. 
This should also include default plugin settings page template.

**Comment by brandonkelly** [comment, marked as answer]
Just went through and added “Save and continue editing” to all settings pages where it was missing, for Craft 4.7 (planned to be released tomorrow). (d46e113f6f77bcd9f5539f35addc7c91d4ccb410)

**Reply by brandonkelly** [reply]
Craft 4.7.0 is now tagged 🚀
